op/requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+collections:
+  - name: community.docker
+    version: '3.4.0'
```

Comment:
it looks like the latest version of `community.docker` collection is `4.4.0`
https://github.com/ansible-collections/community.docker/releases

can you try seeing if that works? if so, let's use the latest version.